composer.json

```diff
@@ -24,7 +24,7 @@
         "illuminate/support": "^9",
         "mockery/mockery": "^1.4.4",
         "phpmyadmin/sql-parser": "^5.5",
-        "phpstan/phpstan": "^1.7.12"
+        "phpstan/phpstan": "1.8.x-dev"
```

Comment:
Isn't 1.8 out?

Reply:
Or we need a recent commit?

Reply:
Yes we need recent commit. So this PR needs to wait until 1.8.1